include/exadg/convection_diffusion/spatial_discretization/operator.cpp

```diff
@@ -551,19 +551,7 @@ template<int dim, typename Number>
 void
 Operator<dim, Number>::interpolate_velocity(VectorType & velocity, double const time) const
 {
-  field_functions->velocity->set_time(time);
-
-  // This is necessary if Number == float
-  typedef dealii::LinearAlgebra::distributed::Vector<double> VectorTypeDouble;
-
-  VectorTypeDouble vector_double;
-  vector_double = velocity;
-
-  dealii::VectorTools::interpolate(get_dof_handler_velocity(),
-                                   *(field_functions->velocity),
-                                   vector_double);
-
-  velocity = vector_double;
+  Utilities::interpolate(get_dof_handler_velocity(), *(field_functions->velocity), velocity, time);
```

Comment:
Is it a mistake that we did not use a mapping here in the old code version?

Reply:
I didn't write the code, so I don't really know why no mapping is used.

Reply:
No problem, I just wanted to ask whether you have had an opinion on this during refactoring.